code/modules/cargo/packs.dm

```diff
@@ -144,7 +144,7 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	name = "FS Handgun Pack"
 	contains = list(/obj/item/weapon/gun/projectile/colt,
 			/obj/item/weapon/gun/projectile/paco,
-			/obj/item/weapon/gun/projectile/clarissa,
+			/obj/item/weapon/gun/projectile/clarissa/clarissa,
```

Comment:
Why the `clarissa/clarissa` ?

Reply:
Cuz else it will force all subtypes to Frozen Star spawn tag
But i think need to be renamed to autopistol/clarissa or etc